packages/scripts/src/admin-proposals/change-final-fee/0_Propose.ts

```diff
@@ -3,13 +3,12 @@
 // To run this on the localhost first fork mainnet into a local hardhat node by running:
 // HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
 // Then execute the script:
-// NEW_FINAL_FEE_USD=<NEW_FINAL_FEE_USD> \
-// NEW_FINAL_FEE_WETH=<NEW_FINAL_FEE_WETH> \
-// UMIP_NUMBER=<UMIP_NUMBER> \
+// PROPOSAL_TITLE=<PROPOSAL_TITLE> \
 // NODE_URL_1=<MAINNET-NODE-URL> \
 // NODE_URL_10=<OPTIMISM-NODE-URL> \
 // NODE_URL_137=<POLYGON-NODE-URL> \
 // NODE_URL_42161=<ARBITRUM-NODE-URL> \
+// TOKENS_TO_UPDATE='{"USDC":{"finalFee":"250.00","mainnet":"0x123","polygon":"0x123","arbitrum":"0x123"}}' \
```

Comment:
Does this script only update if there are changes? Or does it try to apply these whitelists/final fee updates regardless of the current state?

Reply:
It only updates if there are changes:
e.g. [only whitelist if needed](https://github.com/UMAprotocol/protocol/blob/44e85c36255d237e0356c8fd0b5bb6677a951079/packages/scripts/src/admin-proposals/change-final-fee/0_Propose.ts#L92), [only set final fee if needed](https://github.com/UMAprotocol/protocol/blob/44e85c36255d237e0356c8fd0b5bb6677a951079/packages/scripts/src/admin-proposals/change-final-fee/0_Propose.ts#L149)